tests/container.tests.ps1

```diff
@@ -836,10 +836,14 @@ Describe "Push Linux Containers" -Tags 'Linux', 'Push' {
             Set-ItResult -Pending -Because "Image is missing when building using ACR"
         }
 
-        foreach($tag in $tags) {
-            Invoke-Docker -Command push -Params @(
-                    ${tag}
-                ) -SuppressHostOutput
+        if("clearlinux" -imatch $Name -or "parrot" -imatch $Name -or "kalirolling" -imatch $Name)
```

Comment:
We should use the same data from the meta.json here.

Either open an issue to fix it later or fix it now.